src/amo/components/OverallRating.js

```diff
@@ -42,4 +74,38 @@ export class OverallRatingBase extends React.Component {
   }
 }
 
-export default translate({ withRef: true })(OverallRatingBase);
+export const mapStateToProps = (state) => ({
+  authToken: state.auth && state.auth.token,
+});
+
+export const mapDispatchToProps = (dispatch) => ({
+  createRating({ rating, authToken, addonID, versionID }) {
+    const postData = { rating, version: versionID };
+    log.debug('about to call API with', postData);
+    return callApi(
+      {
+        endpoint: `addons/addon/${addonID}/reviews`,
+        body: postData,
+        method: 'post',
+        auth: true,
+        state: { token: authToken },
```

Comment:
We've been passing `state: state.api` in the other calls so that if we need to add more data we can easily add it for all calls. That way the user's current `lang` will also be sent so we can use that (although it doesn't necessarily match the langauge they wrote in).